specification/types.md

```diff
@@ -93,3 +93,13 @@ A structure containing the following fields:
 A structure which supports definition of arbitrary properties, with keys of type `string`, and values of type `boolean`, `string`, or `number`.
 
 This structure is populated by a provider for use by an [Application Author](./glossary.md#application-author) (via the [Evaluation API](./glossary.md#evaluation-api)) or an [Application Integrator](./glossary.md#application-integrator) (via [hooks](./sections/04-hooks.md)).
+
+### Provider Events
+
+An enumeration of provider events.
```

Comment:
Let's emphasize that these are currently accepted/agreed event types.  And to support extensibility as discussed [1], let's include a section to describe how an interested party can make a request to add additional event types through OFEP

[1] - https://github.com/open-feature/spec/pull/182/files#r1178321218

Reply:
I think the spec should be very tight and concise. I'd like to avoid sections in the spec on how to modify it. I've not seen that sort of thing in other specifications, I think it muddies the waters.

Reply:
The idea came from the `IANA Considerations` section and seeing the approach in RFCs (ex: - initial content [1] & registration process [2]) . However, as you said, I am not 100% sure this fully matches our requirements. 

Regardless, I assume there could be new event types of events beyond the initial ones we define here.

[1] - https://www.rfc-editor.org/rfc/rfc6749#section-11.2.2
[2] - https://www.rfc-editor.org/rfc/rfc6749#section-11.3.1